crates/consensus/auto-seal/src/lib.rs

```diff
@@ -287,25 +287,18 @@ impl StorageInner {
         let mut header = Header {
             parent_hash: self.best_hash,
             ommers_hash: proofs::calculate_ommers_root(ommers),
-            beneficiary: Default::default(),
             state_root: Default::default(),
             transactions_root: proofs::calculate_transaction_root(transactions),
             receipts_root: Default::default(),
```

Comment:
@mattsse Here as default values for `state_root` and `receipts_root` shouldn't we set `EMPTY_ROOT_HASH` as it is the case for the `Default` implementation of `Header`?